hadoop-common-project/hadoop-common/src/site/markdown/SecureMode.md

```diff
@@ -20,7 +20,9 @@ Hadoop in Secure Mode
 Introduction
 ------------
 
-This document describes how to configure authentication for Hadoop in secure mode. When Hadoop is configured to run in secure mode, each Hadoop service and each user must be authenticated by Kerberos.
+In its default configuration, we expect you to make sure attackers don't have access to your Hadoop deployment by restricting all network access. If you want to expose Hadoop to untrusted users, you will have to configure authentication for Hadoop in secure mode as described in this document.
```

Comment:
I don't think people cloud deployments realise they've exposed their hadoop services to the world, so we need to be a bit more explicit here.